frontend/src/scenes/sessions/SessionsView.tsx

```diff
@@ -159,22 +147,22 @@ export function SessionsView({ personIds, isPersonPage = false }: SessionsTableP
             title: (
                 <Tooltip
                     title={
-                        currentTeam?.session_recording_opt_in ? (
-                            <>
-                                Replay sessions as if you were in front of your users. Not seeing a recording you're
-                                expecting? <a onClick={() => shareFeedbackCommand()}>Let us know</a>.
-                            </>
-                        ) : (
-                            enableSessionRecordingCTA
-                        )
+                        currentTeam?.session_recording_opt_in
+                            ? 'Replay sessions as if you were right beside your users.'
+                            : enableSessionRecordingCTA
                     }
                 >
                     <span style={{ whiteSpace: 'nowrap' }}>
-                        Recordings
                         {currentTeam?.session_recording_opt_in ? (
-                            <QuestionCircleOutlined className="info-indicator" />
+                            <>
+                                Recordings
+                                <QuestionCircleOutlined className="info-indicator" />
+                            </>
                         ) : (
-                            <PoweroffOutlined className="info-indicator text-warning" />
+                            <Link to={urls.projectSettings()}>
```

Comment:
Would be nice to link directly to the tab. Not sure if it should be:
```jsx
  <Link to={`${urls.projectSettings()}#session-recording`}>
```

... or `urls.projectSettings({ anchor: 'session-recording' })` 

... or any other variation.